classic-editor.php

```diff
@@ -311,7 +314,7 @@ public static function user_settings() {
 		?>
 		<table class="form-table">
 			<tr class="classic-editor-user-options">
-				<th scope="row"><?php _e( 'Editor', 'classic-editor' ); ?></th>
+				<th scope="row"><?php _e( 'Default Editor', 'classic-editor' ); ?></th> <!-- # TWEAK: more detailed action specification Default Editor instead Editor for better understanding -->

```

Comment:
Yeah, may be nice to add "Default" there :)

Reply:
Yeah!